prov/util/src/util_atomic.c

```diff
@@ -629,7 +695,7 @@ size_t ofi_datatype_size(enum fi_datatype datatype)
 		for (i = 0; i < cnt; i++) {				\
 			temp_c = c[i];					\
 			temp_s = s[i];					\
-			(void) op(type, d[i], temp_s, temp_c);		\
+			op(type, d[i], temp_s, temp_c);			\
```

Comment:
nit: extra space at end.  Why was the cast to void removed?  That might have been there to silence some warning from some random code analysis tool.

Reply:
This looks to some sloppiness that climbed in during the debugging process that I didn't clean up for some reason. Sorry. I will back it out.